package.json

```diff
@@ -193,7 +193,7 @@
     "opn": "^5.5.0",
     "rimraf": "2.6.3",
     "shelljs": "^0.8.3",
-    "vscode-azureextensionui": "^0.26.5",
+    "vscode-azureextensionui": "^0.29.0",
```

Comment:
Update to new version of our shared package that supports adding custom info to issues that the user submits from our error popups